lib/models/page.js

```diff
@@ -541,7 +541,7 @@ module.exports = function(crowi) {
     var User = crowi.model('User');
     var limit = option.limit || 50;
     var offset = option.offset || 0;
-    var conditions = { creator: user._id, redirectTo: null };
+    var conditions = { creator: user._id, redirectTo: null, status: STATUS_PUBLISHED };
```

Comment:
- While the field `status` in Page model has appeared since very recent version, `status` should be specified `null or STATUS_PUBLISHED` to find the page with from old version of crowi.
- Pages that has STATUS_DRAFT or some other statues should be displayed on this page (of course, you know some of these statues are not yet implemented though), so how about `status != deleted` ?